views/publication/summary/summary.templ

```diff
@@ -162,26 +162,28 @@ templ Summary(c *ctx.Ctx, args SummaryArgs) {
 								CurrentUserRoles: getUserContributorRoles(args.Publication, c.User),
 							})
 						</div>
-						if args.Publication.VABBID != "" || args.Publication.Legacy || len(args.Publication.RelatedDataset) > 0 {
+						<div class="vstack gap-3">
+							<div class="d-flex align-items-center">
+								<i class="if if-building if--small if--muted me-2"></i>
+								if len(args.Publication.RelatedOrganizations) > 0 {
+									// TODO: pass on args.Target
+									@relatedorganizationviews.Summary(c, args.Publication.RelatedOrganizations, views.URL(args.URL).SetQueryParam("show", "contributors").String())
+								} else {
+									<a
+										href={ views.URL(args.URL).SetQueryParam("show", "contributors").SafeURL() }
+										class="c-link-muted"
+										if args.Target != "" {
+											target={ args.Target }
+										}
+									>
+										<i class="if if-edit if--small"></i>
+										<em>Add department</em>
+									</a>
+								}
+							</div>
+						</div>
+						if args.Publication.VABB() != "" || args.Publication.Legacy || len(args.Publication.RelatedDataset) > 0 {
```

Comment:
I can only add VABB year via batch (works as intended), I cannot fake information such as `VABB: c:vabb:495300, VABB-1, approved, 2021`. Will this piece of code also show that information when available, and not just the year?

Reply:
yes args.Publication.VABB() will should all the info as a string